inc/link.class.php

```diff
@@ -464,6 +463,21 @@ public function showOrderLinkItem($numref, $data_ref, $canedit, $plugin_order_or
       echo "<br>";
    }
 
+   public function getSerialItem($data){
+      global $DB;
+      $itemtype = strtolower($data['itemtype']);
+      $itemtype_s = $itemtype."s";
+      if ($itemtype_s!="pluginorderothers") {
```

Comment:
`PluginOrderReferenceFree` items does not have serial field, should be excluded too.